test_app/tests/conftest.py

```diff
@@ -14,6 +14,29 @@
 from test_app import models
 
 
+def copy_fixture(copies=1):
```

Comment:
This looks like it was copied from here?

https://github.com/ansible/django-ansible-base/blob/5d9db2d0f5470beb1b6fd115ad219d649095a80d/ansible_base/lib/testing/util.py#L1

You could import it if you needed it.

Reply:
oh, fantastic! I copied it from the gateway repo haha. I didn't realize it was in ansible base